src/utils/installPackage.js

```diff
@@ -0,0 +1,51 @@
+const {spawn} = require('child_process');
+const config = require('./config');
+
+module.exports = async function(dir, name) {
+  let yarn;
+  try {
+    yarn = await config.resolve(dir, ['yarn.lock']);
+  } catch (e) {}
+
+  return new Promise((resolve, reject) => {
+    let options = {
+      cwd: dir
+    };
+
+    let install = spawn('yarn', ['add', name, '--dev'], options);
```

Comment:
should they be dev dependencies or normal dependencies? I could see arguments for both.

Reply:
I think because it's a bundler it should only happen at development not sure though